src/source/PeerConnection/SessionDescription.c

```diff
@@ -757,10 +759,6 @@ STATUS reorderTransceiverByRemoteDescription(PKvsPeerConnection pKvsPeerConnecti
         }
     }
 
-    // delete the unordered part in pKvsPeerConnection->pTransceievers
```

Comment:
Are we sure we need to remove these? If my reading of the code is right (and I am rusty here) the ordered part will be copied at the tail so the front might need to be removed.

Reply:
Also, it doesn't let me comment on LN 697 RTC_CODEC rtcCodec; - static tooling complains that this might be uninitialized. We need to either MEMSET it or initialize with {0}

Reply:
> Are we sure we need to remove these? If my reading of the code is right (and I am rusty here) the ordered part will be copied at the tail so the front might need to be removed.

i deleted these in `copyTransceiverWithCodec` - no need to delete again

before:
1. transceivers = `unsorted: transceiver2, transceiver1, sorted: transceiver1, transceiver2`
2. delete 2 transceivers
3. transceivers = `sorted: transceiver1, transceiver2`

after (with the modifications in this PR):
1. transceivers = `sorted: transceiver1, transceiver2`

Reply:
> Also, it doesn't let me comment on LN 697 RTC_CODEC rtcCodec; - static tooling complains that this might be uninitialized. We need to either MEMSET it or initialize with {0}

i noticed it as well, but there's no `RTC_CODEC_UNKNOWN` enum, i think it should be done in a separate PR.